src/ontology/edits/oeo-social.omn

```diff
@@ -677,16 +679,13 @@ pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1558",
     
 Class: OEO_00010079
 
-    SubClassOf: 
-        OEO_00020056 some OEO_00020063
```

Comment:
This axiom of a shared class was implemented in oeo-social and not in oeo-shared. Therefore I moved it to oeo-shared-axioms.